perceval/backend.py

```diff
@@ -253,6 +267,10 @@ def metadata_updated_on(item):
     def metadata_category(item):
         raise NotImplementedError
 
+    @staticmethod
+    def search_terms(item):
```

Comment:
Would it be better to use `search_fields` instead of `search_terms`? I'm saying this because we have `classified_fields`, so maybe it makes more sense to align it with that concept too.